From 4aac4b73d5c334f95a7850874847df463c1f5874 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Thu, 2 Aug 2018 09:40:34 +0800
Subject: [PATCH] arm: dts: enable dcan by default

DCAN will be enabled in profile #1, but it's safe to be okay it by default.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
Signed-off-by: Xiaolei Wang <xiaolei.wang@windriver.com>
---
 arch/arm/boot/dts/am335x-evm.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/am335x-evm.dts b/arch/arm/boot/dts/am335x-evm.dts
index a00145705c9b..67bc672e6360 100644
--- a/arch/arm/boot/dts/am335x-evm.dts
+++ b/arch/arm/boot/dts/am335x-evm.dts
@@ -806,7 +806,7 @@
 };
 
 &dcan1 {
-	status = "disabled";	/* Enable only if Profile 1 is selected */
+	status = "okay";	/* Enable only if Profile 1 is selected */
 	pinctrl-names = "default";
 	pinctrl-0 = <&dcan1_pins_default>;
 };
-- 
2.17.1

